### PR TITLE
Verified Chartboost iOS adapter compatibility with Chartboost SDK 8.1.0

### PR DESCRIPTION
**Pull request by joshuajplagonera**
Verified Chartboost iOS adapter compatibility with Chartboost SDK 8.1.0

### CODE REVIEW

File: adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboost.h [hunk 1]
```diff
@@ -17,12 +17,7 @@
 #import <Foundation/Foundation.h>
```
Comment (daniel-barros) [inline adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboost.h:17]: Can we remove the two Chartboost imports above?

File: adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostBannerAd.m [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterChartboostBannerAd.h"
+
+#import <Chartboost/Chartboost+Mediation.h>
```
Comment (daniel-barros) [inline adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostBannerAd.m:17]: Can we substitute this import by this?
```
#if __has_include(<Chartboost/Chartboost+Mediation.h>)
#import <Chartboost/Chartboost+Mediation.h>
#else
#import "Chartboost+Mediation.h"
#endif
```
It helps with our testing process by making mocking Chartboost headers easier.

**Comment by daniel-barros** [comment]
Hi @joshuajplagonera 
Could you take a look at https://github.com/googleads/googleads-mobile-ios-mediation/pull/227?

**Comment by joshuajplagonera** [comment]
> Hi @joshuajplagonera
> Could you take a look at #227?

Hi @daniel-barros 
I pulled in your changes from #227 and updated this PR. Please take a look.

**Comment by daniel-barros** [comment]
Hi @joshuajplagonera, thanks for your updates!
We've identify only two remaining issues: a bug that causes Chartboost extras to no being handled correctly, and a possible race condition due to the initialization adapter not being thread-safe.

I've prepared two PRs with our suggested changes to solve these issues.

The first one also includes changes on how Chartboost headers are imported, making the adapters compatible with our internal testing tools we use for other mediations at Chartboost. These changes were already proposed and applied in some files but not in others, we'd like to make a consistent use of the proposed new method.
This is the PR: https://github.com/googleads/googleads-mobile-ios-mediation/pull/233

The second one includes fixes for the issues mentioned but no changes to the imports.
https://github.com/googleads/googleads-mobile-ios-mediation/pull/232

**Comment by joshuajplagonera** [comment]
Hi @daniel-barros ,
Thank you for taking a look at the PR. Could you let us know if this PR looks good to merge? 

Instead of updating this PR with your suggestions and avoid a larger diff, is it okay to merge this PR and you can update #233 and resolve any conflicts, if any, and we can merge that PR ?

**Comment by daniel-barros** [comment]
Sounds good to me @joshuajplagonera, thanks!

**Pull request merged**